### DISCUSSION TITLE
Native whitebalance

### DISCUSSION THREAD

**Discussion by sanj3k** [gopro/labs#278] [General]
Hi, long time ago when I just started flying gopros on fpv drones it was suggested to always use native setting to have more information etc. but most of the times the results had a orange-ish tone which I couldnt get rid at all. At some point in time I have stumbled upon xtremestuff/protune-transforms and even years after applying it now it magically helps to get the real color back. 

So my first question is, is this still valid and not outdated? (I used it on H6 footage but the question is more towards the H10/H11) 
Are the any other similar correction possibilities or how do you work with native white balance in general? 
And at what temperatures start some limitations? Because setting native would be less of a headache than sweeping through and deciding for a fixed wb value for me.

**Comment by dnewman-gpsw** [comment]
Consider !MWIDE=1 rather then native, it has a similar gamut without the color bias, and it work in a wider range of lighting conditions.

HERO11 only see https://gopro.github.io/labs/control/extensions/
